File: world/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
    int height;

    // Prompt the user for the height
    do {
        height = get_int("Height: ");
    } while (height <= 0);

    // Draw the pyramid
    for (int i = 0; i < height; i++) {
        // Print the spaces
        for (int j = 0; j < height - i - 1; j++) {
            printf(" ");
        }

        // Print the hashes
        for (int k = 0; k <= i; k++) {
            printf("#");
        }

        printf("\n");
    }

    return 0;
}

File: sentimental-mario-less/mario.py

def get_int(prompt):
    while True:
        try:
            value = int(input(prompt))
            if 1 <= value <= 8:
             return value
        except ValueError:
            continue

def draw_piramid():
    height = get_int("Enter the height of the pyramid: ")

    for i in range(1, height + 1):
        print(" " * (height-1) + "#" * i, end="")
        print()

draw_piramid()
